themes/src/main/resources/theme/base/admin/resources/js/controllers/realm.js

```diff
@@ -965,6 +965,7 @@ module.controller('RealmIdentityProviderCtrl', function($scope, $filter, $upload
         for (var key in data) {
             $scope.identityProvider.config[key] = data[key];
         }
+        $scope.identityProvider.enabled = data["enabled"];
```

Comment:
This change is not needed.

Reply:
Related to my comment in SAMLIdentityProviderConfig, data returned from rest api return only Map. "enabled" key containing in data Map will never affect identityProvider.enabled with previous code. Do you suggest another solution?

My current implementation need a change in order to take into account that "enabled" key will not exist in Map for all other Identity providers except than SAML Identity Provider. I proposed to be : 
`if (data["enabled"] != undefined) {
        	$scope.identityProvider.enabled = data["enabled"];
        }`